src/rdkafka_conf.c

```diff
@@ -1121,10 +1128,29 @@ static int rd_kafka_conf_s2i_find (const struct rd_kafka_property *prop,
 
 static rd_kafka_conf_res_t
 rd_kafka_anyconf_set_prop (int scope, void *conf,
-			   const struct rd_kafka_property *prop,
-			   const char *value,
-			   char *errstr, size_t errstr_size) {
+                           const struct rd_kafka_property *prop,
+                           const char *value,
+                           char *errstr, size_t errstr_size) {
 	int ival;
+    
+	/* Check if configuration scopes match */
+	int* conf_scope = (scope & _RK_GLOBAL) ? &((rd_kafka_conf_t*)conf)->scope :
```

Comment:
Please read the coding guidelines:
 * pointer goes with the variable name, not type
 * <= 80 columns

Reply:
ok